doc/web/content/Download/Binaries/Linux.html

```diff
@@ -21,25 +21,25 @@ <h2><img class="blogo" src="images/logo_linux35.png"
            alt="[Tux]"/>Linux</h2>
   <p class="first"> The package:
     <code><a type="application/x-executable"
-             href="https://github.com/tlaplus/tlapm/releases/latest/download/tlaps-1.4.5-x86_64-linux-gnu-inst.bin">tlaps-1.4.5-x86_64-linux-gnu-inst.bin</a></code>
+             href="https://github.com/tlaplus/tlapm/releases/download/202210041448/tlaps-1.5.0-x86_64-linux-gnu-inst.bin">tlaps-1.5.0-x86_64-linux-gnu-inst.bin</a></code>
```

Comment:
In the (near) future when we switch to a rolling release model I suppose we will want to use the /latest/ tag along with a link to a binary that does not have the version in the name.

Reply:
I agree. This will have to be changed in the future.
For now, I just wanted to keep links valid after a new release is made.